go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/aws/aws-xray-daemon
+
+go 1.13
+
+require (
+	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
+	github.com/aws/aws-sdk-go v1.25.42
```

Comment:
Hello @jasonrichardsmith, 
Sorry for this PR not getting prioritized. We will be reviewing it as soon as we have bandwidth. In the meantime, could you please update the version of the `aws-sdk-go` module to update that in PR #42? I plan on closing that PR in favor of this one, but the upgrade must happen for internal testing issues.